scipy/stats/_stats_py.py

```diff
@@ -2845,7 +2845,10 @@ def obrientransform(*samples):
     return np.array(arrays)
 
 
-def sem(a, axis=0, ddof=1, nan_policy='propagate'):
+@_axis_nan_policy_factory(
+    lambda x: x, result_to_tuple=lambda x: (x,), n_outputs=1, too_small=1
+)
+def sem(a, axis=0, ddof=1):
```

Comment:
```suggestion
def sem(a, axis=0, ddof=1, nan_policy='propagate'):
```
I think we need to leave this because `nan_policy` still needs to be accepted as a positional arg.

Reply:
Ah ok I thought the decorator was adding it as part of the overwrite with the doc too.

Reply:
It does add it if it is not already there, but it would add it as a keyword-only arg unless this is already there as a positional arg.

Reply:
I see, so backward incompatible potentially.